book.md

```diff
@@ -1099,11 +1099,48 @@ first, it is called a side-channel attack. The entity not intending to
 communicate is called the victim\index{victim}. The other entity is sometimes
 called the spy\index{spy}.
 
-The rest of this chapter mostly describes a variety of common covert channel
-mechanisms. It does not aim to differentiate much on whether both ends intend
-to cooperate, or whether one end is a victim under attack of the other end.
+There are a lot of covert channels and side-channels. We describe the most
```

Comment:
is the use of `covert` here ok? Are the terms `covert channels` and `side channels` interchangable? I noticed you changed an instance of the former to the latter in line 1256. Might be a good opportunity to make it consistent in this paragraph but only if you think that makes sense from a technical perspective.

Reply:
Thank you for the review and pointing this out!
I've searched for the term "covert" in the book and adapted text where it didn't make sense anymore.
There are still a few occurrences of "covert" left where I thought it made more sense to used the term "covert channel" than "side-channel".